resources/cross/wasm.txt

```diff
@@ -0,0 +1,42 @@
+# cross file for WASM.
```

Comment:
wasm builds are non native, `needs_exe_wrapper` needs to be set so meson gets no silly ideas about what we can run
e.g.

```
[properties]
needs_exe_wrapper = true
```